components/callbacks/executors.go

```diff
@@ -54,21 +54,18 @@ type HTTPCaller func(*http.Request) (*http.Response, error)
 func RegisterExecutor(
 	registry *hsm.Registry,
 	activeExecutorOptions ActiveExecutorOptions,
-	standbyExecutorOptions StandbyExecutorOptions,
 	config *Config,
 ) error {
 	activeExec := activeExecutor{options: activeExecutorOptions, config: config}
-	standbyExec := standbyExecutor{options: standbyExecutorOptions}
 	if err := hsm.RegisterImmediateExecutor[InvocationTask](
 		registry,
 		activeExec.executeInvocationTask,
 	); err != nil {
 		return err
 	}
-	return hsm.RegisterTimerExecutors[BackoffTask](
+	return hsm.RegisterTimerExecutor[BackoffTask](
 		registry,
 		activeExec.executeBackoffTask,
```

Comment:
We can rename all of these executors to just be called `executor` now that there's no notion of `standby`.

Reply:
I'll is a separate PR

Reply:
Renamed everything in this PR.